src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue29693.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue29693 : _IssuesUITest
+{
+	public override string Issue => "The default native on color is not displayed when the Switch on color is not explicitly set";
+
+	public Issue29693(TestDevice device)
+	: base(device)
+	{ }
+
+	[Test]
+	[Category(UITestCategories.Switch)]
+	public void VerifySwitchDefaultColors()
+	{
+		App.WaitForElement("button1");
+		App.Tap("button1");
+		App.Tap("button2");
+		VerifyScreenshot();
```

Comment:
Pending snapshots already available in the latest build. Could you commit the images?
Example:
![image](https://github.com/user-attachments/assets/0643bbfa-9f37-4b30-87ac-37504642dc21)